.circleci/config.yml

```diff
@@ -251,6 +251,22 @@ jobs:
             # assert inspector installed
             pip show dwave-inspectorapp
 
+  test-asv:
+    docker:
+      - image: cimg/python:3.11
+
+    steps:
+      - checkout
+
+      - run:
+          name: Verify benchmarks run
+          command: |
+            python -m venv env
+            . env/bin/activate
+            pip install -r benchmarks/requirements.txt
+            asv machine --yes
+            asv run -q
```

Comment:
These are used here as plain functionality tests, right, and not for any actual performance degradation detection?

Reply:
Exactly, just a functionality test in CI.